module.yml

```diff
@@ -103,6 +103,14 @@ Parameters:
     Default: 80
     MinValue: 1
     MaxValue: 49150
+  AppEnvironmentVariables:
+    Description: 'An optional string representation of a list of environment variables. Each key value pair is separated by ,, and each key and value by ==. eg. "key1==value1,,key2==value2"'
+    Type: String
+    Default: 'END_MARKER'
+  AppSecretVariables:
+    Description: 'An optional CommaDelimitedList of up to 10 Environment Variables with the value the name of a secret module. Eg. Key1=SecretModuleName'
+    Type: CommaDelimitedList
+    Default: 'END_MARKER'
```

Comment:
wouldn't the default `''` work as well?

Reply:
I'll check that out. I certainly needed this Default value for some reason but there have been a few changes so perhaps not.